applications/RANSApplication/tests/cpp/k_epsilon/test_utilities.h

```diff
@@ -0,0 +1,42 @@
+//    |  /           |
```

Comment:
Same comment to ease going through the code. I suggest renaming to `k_epsilon_test_utilities.h`.